core-services/ci-secret-generator/_config.yaml

```diff
@@ -158,6 +158,10 @@
     - github-ldap-user-group-creator
     - hive
 - fields:
+  - cmd: oc --kubeconfig /tmp/build-farm-credentials/sa.config-updater.$(cluster).config
+      create token -n clusters hypershift-workload-sa --duration=2419200s --bound-object-kind=Secret
```

Comment:
Is the service account named `hypershift-workload-sa`?
That sounds odd.
The naming convention is something like `deck` for the component deck, instead of `deck-sa`.
Could we name it `hypershift-workload`, or something like it?

Reply:
we can fix it